docs/DeveloperGuide.adoc

```diff
@@ -47,6 +47,8 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 |`* *` |user |hide <<private-contact-detail, private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user with different types of contacts |categorise my contacts |search for relevant contacts more easily
```

Comment:
You could choose to provide a `find` option rather than categorize to search for relevant contacts, no?